tests/Backend/CommonPart1/ImportExportCommonTest.php

```diff
@@ -404,6 +405,21 @@ public function testTableAsyncImportMissingFile(): void
         }
     }
 
+    public function testImportAsNull(): void
```

Comment:
udelej tady prosim jeste test na to, kdyz tam posles vice hodnot v `importAsNull`

Reply:
Done